abis_mapping/utils/rdf.py

```diff
@@ -31,6 +31,7 @@
     ("geo", namespaces.GEO),
     ("tern", namespaces.TERN),
     ("dwc", namespaces.DWC),
+    ("bdra", namespaces.BDRA),
```

Comment:
@joecrowleygaia , could change the namespace to be just bdr/BDR (not BDRA). I've told Mieke to change it but it must have slipped her mind.

Reply:
@serge-gaia should i be changing the uri of the namespace from `http://example.com/bdra-schema/` to `http://example.com/bdr-schema/`

Reply:
yes

Reply:
It's just a placeholder namespace that we, Gaia, use when we make things up. For example a new Class that doesn't exist in the Tern schema but that we think should exist.

Reply:
changed both URI and prefix lettering